src/change_stream/option.rs

```diff
@@ -0,0 +1,86 @@
+use serde::{Deserialize, Serialize};
+use std::time::Duration;
+use typed_builder::TypedBuilder;
+
+use crate::{
+    change_stream::document::ResumeToken,
+    collation::Collation,
+    options::AggregateOptions,
+};
+
+#[derive(Clone, Debug, Default, Deserialize, Serialize, TypedBuilder)]
+#[serde(rename_all = "camelCase")]
+pub struct ChangeStreamOptions {
+    /// When set to "updateLookup", the `full_document` field of the `ChangeStreamEventDocument`
```

Comment:
It would be a little clearer to put `FullDocumentType::UpdateLookup` here instead of using quotations

Reply:
makes sense. fixed

Reply:
fixed